.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.cs]
+file_header_template = --------------------------------------------------------------- \nCopyright (c) MumsWhoCode. Learn,. All rights reserved. \n---------------------------------------------------------------
```

Comment:
The message is "Copyright (c) MumsWhoCode. All rights reserved."

Reply:
Ensure you are using the right tag for the commit messages:
<img width="230" alt="image" src="https://user-images.githubusercontent.com/89320816/143148180-bf2fd9a4-26e2-4043-bfd9-14fbfe160406.png">